models/models_methods.go

```diff
@@ -288,6 +288,15 @@ func (workflow *WorkflowModel) Validate() ([]string, error) {
 			return warnings, err
 		}
 
+		stepIDData, err := CreateStepIDDataFromString(stepID, defaultStepLibSource)
```

Comment:
pls solve to replace this function, as the new input (defaultStepLibSource), is only needed for using the `CreateStepIDDataFromString`, but we do not need the StepSource information otherwise